src/components/ZetkinSpeedDial/actions/createEvent.tsx

```diff
@@ -5,6 +5,7 @@ import { Event } from '@material-ui/icons';
 import postEvent from '../../../fetching/postEvent';
 import { useMutation } from 'react-query';
 import { useRouter } from 'next/router';
+
```

Comment:
How did this little thing make it in? 😉

Reply:
I like to separate the imports that are from libraries,and from our codebase. It's just my style 😎

Sometimes I separate imports from the same folder and from far up the component tree. To me it's just cleaner and easier to orient myself in that.

Reply:
deleted my last comment. Will fix and reorder!

Reply:
Just a note: I agree with separating library from "inhouse" imports. 😊